### ISSUE TITLE
Debugger not populating values for Object instances created from a custom constructor function.

### ISSUE THREAD

**Issue by ghgofort** [SqrTT/prophet#82]
## Summary:
When I create a constructor function for instantiating Object instances, the instances appear as [Object, Object] in the debugger, with no option for expanding. I am able to get to the properties of the object from the debugger CLI, but this is not ideal.

## Example:
- Test Controller:

```javascript
function Car(args) {
  this.year = args && args.year ? args.year : 1999;
  this.make = args && args.make ? args.make : 'No Value';
  this.model = args && args.model ? args.model : 'No Value';
}

exports.TestDebugger = function() {
  const myCar = new Car({year: 2012, make: 'Honda', model: 'Pilot'});

  response.setContentType('application/json');
  response.writer.print(JSON.stringify(myCar));
};

exports.TestDebugger.public = true;
```

If I add a breakpoint to the code after the creation of the car instance all I am able to inspect from the myCar variable is [object, Object]:

![image](https://user-images.githubusercontent.com/16811151/36704499-dc94457c-1b1d-11e8-8357-2642858220ff.png)

Another example with the function on the same scope level:

![image](https://user-images.githubusercontent.com/16811151/36704769-02526e64-1b1f-11e8-9620-d02dcf189d89.png)

I am able to access the properties using the CLI but this is less than ideal with more complex Object instances. I would very much like to be able to view these objects in the variables panel along with the other variables.

I am more than willing to help with implementing this functionality if it is something that you would consider?

Thanks for considering, Galen.

**Comment by SqrTT** [comment]
Hi, @ghgofort 

Yes, this is definitely a bug and can be fixed.

thanks for reporting.

**Comment by SqrTT** [comment]
Hi @ghgofort 
the issue should be fixed in `v0.10.3`. Could you please confirm?

**Comment by ghgofort** [comment]
@SqrTT - Confirmed as fixed.  I am closing since I opened. Thx for addressing.
